packages/turbo-workspace-convert/src/cli.ts

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env node
+
+import inquirer from "inquirer";
+import path from "path";
+import meow from "meow";
+
+import {
+  isYarnAvailable,
+  isPnpmAvailable,
+  isNpmAvailable,
+  PackageManagerAvailable,
+} from "turbo-utils";
+import getWorkspaceDetails from "./getWorkspaceDetails";
+import convert from "./convert";
+import { PackageManagers } from "./types";
+import { Logger } from "./logger";
+
+// find all available package managers
+const availablePackageManagers: Record<
+  PackageManagers,
+  PackageManagerAvailable
+> = {
+  yarn: isYarnAvailable(),
+  pnpm: isPnpmAvailable(),
+  npm: isNpmAvailable(),
+};
+
+function isPackageManagerDisabled({
+  packageManager,
+  currentWorkspaceManger,
+}: {
+  packageManager: PackageManagers;
+  currentWorkspaceManger: PackageManagers;
+}) {
+  if (currentWorkspaceManger === packageManager) {
+    return "already in use";
+  }
+
+  if (!availablePackageManagers[packageManager].available) {
+    return "not installed";
+  }
+
+  return false;
+}
+
+interface Answers {
+  packageManager: PackageManagers;
+}
+
+const help = `
+  Usage:
+    $ npx @turbo/workspace-convert [flags...] [<dir>]
+
+  If <dir> is not provided, you will be prompted for it.
+
+  Flags:
+    --npm           Switch to npm workspaces
+    --pnpm          Switch to pnpm workspaces
+    --yarn          Switch to yarn workspaces
+    --install       Convert lock files and install dependencies
+    --dry           Run without making any modifications
+    --summary       Show a summary of the workspace at <dir>, including the detected package
```

Comment:
Hm, maybe I'm really just asking for there to be a `convert` subcommand? `summary` could be a no-args, no-flags kind of deal.

Maybe the package name could be `workspace-inspector`?